wallet/package.json

```
@@ -83,9 +83,10 @@
     "build": "yarn build:ses && yarn build:react",
     "build:ses": "cp node_modules/ses/dist/lockdown.umd.js public/",
     "build:react": "react-app-rewired build",
-    "format": "yarn prettier --write .",
+    "pretty-check": "yarn prettier --check '**/*.{js,jsx,ts,tsx}'",
+    "pretty-fix": "yarn prettier --write '**/*.{js,jsx,ts,tsx}'",
```

Please align these commands cwith agoric-sdk:
```suggestion
    "lint:format": "yarn prettier --check '**/*.{js,jsx,ts,tsx}'",
    "format": "yarn prettier --write '**/*.{js,jsx,ts,tsx}'",
```

"format" instead of "pretty" because we could change formatting tools and wouldn't want to reprogram muscle memory. "lint:format" so that the `run-s` in `lint` does that one too.

---

Done